binr/radare2/radare2.c

```diff
@@ -1176,7 +1176,7 @@ int main(int argc, char **argv, char **envp) {
 #endif
 #endif
 	if (fullfile) {
-		r_core_block_size (&r, r_io_desc_size (r.io, r.file->desc));
+		r_core_block_size (&r, r_io_desc_size (r.file->desc));
 	}
```

Comment:
yes, it is inside RIODesc, but that is not the point here. RIOPlugin is inside RIODesc, too. For calculating the size, you don't need the io, if you have the desc and the plugin. So clean alternative would be r_io_size (io, fd), and having desc_size for internal use mostly